modules/mono/glue/GodotSharp/GodotSharp/Core/Colors.cs

```diff
@@ -171,12 +172,14 @@ public static class Colors
         public static Color Blue { get { return namedColors["blue"]; } }
         public static Color BlueViolet { get { return namedColors["blueviolet"]; } }
         public static Color Brown { get { return namedColors["brown"]; } }
-        public static Color BurlyWood { get { return namedColors["burlywood"]; } }
+        public static Color BurlyWood { get { return namedColors["burlywood"]; } } // for compatibility
+        public static Color Burlywood { get { return namedColors["burlywood"]; } }
         public static Color CadetBlue { get { return namedColors["cadetblue"]; } }
         public static Color Chartreuse { get { return namedColors["chartreuse"]; } }
         public static Color Chocolate { get { return namedColors["chocolate"]; } }
         public static Color Coral { get { return namedColors["coral"]; } }
-        public static Color Cornflower { get { return namedColors["cornflower"]; } }
+        public static Color Cornflower { get { return namedColors["cornflower"]; } } // for compatibility
+        public static Color CornflowerBlue { get { return namedColors["cornflowerblue"]; } }
```

Comment:
Good catch with `CornflowerBlue` and `LimeGreen`, those are the names in both the X11 and .NET color names. However, we don't need to keep the old names, we can break compatibility for 4.0.

Reply:
I observed all the PRs related to the breaking compatibility bits, so I'll comply.

Reply:
What about the ones in [color_names.inc]?(https://github.com/godotengine/godot/pull/41019/files/3b118080217bddafee350aeb6cb37ac687e6f7ce#diff-6caad16a9471f9f3b427c06ce47a9b78)